src/androidTest/java/com/nutomic/syncthingandroid/test/syncthing/RestApiTest.java

```diff
@@ -75,7 +73,7 @@ public void onReceiveSystemInfo(RestApi.SystemInfo info) {
     public void testGetFolders() {
         assertNotNull(mApi.getFolders());
     }
-    
+
     public void testConvertNotCrashing() {
```

Comment:
Hmm, as expected, I had had conflicts upon rebase and I then accepted the version that came from you without any further editing.